change/@fluentui-react-tree-28a20b5c-2da7-46ae-9716-fe0064f96cd4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29338&commitId=b97e5dd5bfa1fa007ff9146b27fbb9a65199bb92&env=prod&clientType=FLUENTUI)


  <details><summary><b>Tree</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.appearance.Subtle tree mousedown.chromium.png|4968| Changed|
|Tree.appearance - RTL.Subtle tree mousedown.chromium.png|4968| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>